Recommend using MSTest SDK for MSTest runner (#40379)
dotnet/docs: docs/core/testing/unit-testing-mstest-runner-intro.md

## Enable MSTest runner in a MSTest project

> We recommend using [MSTest SDK](./unit-testing-mstest-sdk.md) as it greatly simplifies your project configuration, the update of the project and ensures a proper alignment of the versions of the platform (MSTest runner) and its extensions.

Reviewer: ```suggestion
It's recommended to use [MSTest SDK](./unit-testing-mstest-sdk.md) as it greatly simplifies your project configuration, the updating of the project and ensures a proper alignment of the versions of the platform (MSTest runner) and its extensions.
```
